python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -426,7 +439,7 @@ async def kiloarcListenerLoop(self):
         except ConnectionException:
             await self.reconnect_kiloarc_or_fault()
 
-        while self.kiloarcListenerLoopBool:
+        while True:
             # if we lose connection to the ADAM, stop loops and go to FAULT
```

Comment:
This comment appears to be outdated. if you lose connection you call `reconnect_kiloarc_or_fault`. I assume that raises an exception if you into fault state, but returns if you regain the connection.